Automattic/maintenance-mode-wp
# Enable via UI rather than code

Fixes #37 

**Description**

Allow users to activate the maintenance mode via customizer in addition to the existing activation option via adding `define( 'VIP_MAINTENANCE_MODE', true )` to functions.php (WordPress.com) resp. to vip-config.php (VIP Go).

**Steps to test**

1. Check out [the featured branch](https://github.com/Automattic/maintenance-mode-wp/tree/update/%2337)
2. Install and activate the plugin on a testing site
3. Go to the frontend of the testing site to ensure that the maintenance mode is not active
4. Go to `WP Admin → Appearance → Customize → Maintenance Mode` and activate the maintenance mode
5. Go to the frontend of the testing site to ensure that the maintenance mode is active

**Screenshots**
<table>
<tr>
<td>

![mm-01](https://user-images.githubusercontent.com/3323310/107119881-f8aa9000-68bc-11eb-8db6-bf617f2705aa.png)
</td>
<td>

![mm-03](https://user-images.githubusercontent.com/3323310/107119929-360f1d80-68bd-11eb-89ea-bcc292b0eaac.png)
</td>
<td>

![mm-04](https://user-images.githubusercontent.com/3323310/107119932-37d8e100-68bd-11eb-899c-3a0fde8ffa31.png)
</td>
</tr>
</table>
